### PR TITLE
GHC 8.8

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

**Comment by neongreen** [comment]
Huh, why four Travises?

File: shower.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: shower
-version: 0.2.0.1
+version: 0.2.0.2
```
Comment (int-index) [inline shower.cabal]: Since there are no code changes, I would avoid bumping the minor version and create a new revision instead.

Comment (neongreen) [reply shower.cabal]: I don't trust the revisions mechanism anymore after seeing reports like https://github.com/monadfix/microlens/issues/112#issuecomment-495449076. I don't have a particularly strong opinion though.

Comment (int-index) [reply shower.cabal]: I see – `nix` can't deal with revisions, apparently. OK.

**Pull request merged**